src/client/resources/assets/snapper/lang/pt_br.json

```diff
@@ -0,0 +1,42 @@
+{
+  "key.snapper.panorama": "Faça Capturas de Tela panorâmicas",
+  "key.snapper.huge": "Faça Captura de Tela enorme",
+  "key.snapper.screenshot_menu": "Exibir Capturas de Tela",
+  "key.snapper.recent": "Abrir a última Captura de Tela",
+  "menu.snapper.screenshotmenu": "Capturas de Tela",
+  "menu.snapper.viewermenu": "Visualizador",
+  "menu.snapper.panorama": "Exibir Panorama",
+  "button.snapper.folder": "Abrir Pasta",
+  "button.snapper.panorama.tooltip": "Visualizar o panorama atual armazenado.",
+  "button.snapper.screenshots": "Exibir Capturas de Tela",
+  "button.snapper.delete": "Excluir",
+  "button.snapper.open": "Abrir Imagem",
+  "button.snapper.view": "Exibir Imagem",
+  "button.snapper.copy": "Copiar",
+  "button.snapper.rename": "Renomear",
+  "text.snapper.created": "Criado",
+  "text.snapper.generic": "Captura de Tela",
+  "text.snapper.loading": "Carregando Capturas de Tela",
+  "text.snapper.empty": "Faça uma Captura de Tela no jogo para vê-la aqui!",
+  "text.snapper.panorama_encourage": "Faça um panorama no jogo para vê-lo aqui!",
+  "text.snapper.panorama_success": "Panorama salvo. Visualize pressionando %s",
+  "text.snapper.panorama_failure_iris": "Devido a um problema no Iris, os panoramas não podem ser tirados enquanto ele estiver instalado.",
```

Comment:
Iris shouldn't need to be referred as "ele". This currently says "the panoramas can't be taken while it's* installed", which isn't wrong but would be clearer if referred to as "o mod", which means "the mod" and would work in the sentence as "the panoramas can't be taken while the mod is installed"

*it's technically "he", but that's only since portuguese doesn't have neutral pronouns. Using he/she in this context can sound more confusing to some people.

Reply:
This suggestion is completely wrong because in Portuguese there is already a neuter pronoun, which is the masculine pronoun.

Reply:
The male pronoun by definition is not neutral. It is commonly used to refer to things that don't have a gender (for example: "A Caixa" ("The (female) box") and "O carro" ("The (male) car")), but this doesn't mean it's neutral.

Recently, some people have started using "elu/delu" as neutral pronouns, and have words like "Todos" (male) be "Todes" (neutral), but it's not officially part of any variation of the Portuguese language and many people don't consider it "valid"

Reply:
So, since Portuguese came from Latin, there used to be a neuter pronoun, but then they switched to the masculine pronoun.

Reply:
What does this prove? Portuguese still does not have any truly neutral pronouns. It genders genderless objects.

Reply:
This proves that it does exist, and that there are other forms such as “isto”, “aquilo". Which are masculine forms too.

Reply:
How does it prove it exists? It's literally gendering genderless objects. A neutral pronoun is something used to refer to a genderless object as genderless. Portuguese doesn't have such a thing, and we default to male instead of using an actual neutral pronoun.

You could refer to Iris as "o mod", which still isn't neutral, but instead is gendering what Iris is as a common noun, instead of gendering Iris itself.

Reply:
If this sounded confusing or wrong, let me try to explain what I'm saying.

Iris is a mod, and a mod is a common noun. Countless mods exist, but each have their own unique name, which are proper nouns.

Think of it like humans. Billions of humans exist, and each has their own unique name. They don't stop being "a human" just because they're named "Fulano" ou "Cicrano"